bliss/simulator/prior.py

```diff
@@ -193,11 +176,11 @@ def _draw_truncated_pareto(self, alpha, min_x, max_x, n_samples) -> Tensor:
         return min_x / (1.0 - uniform_samples) ** (1 / alpha)
 
     def _sample_star_fluxes(self, star_ratios):
-        latent_dims = (self.batch_size, self.n_tiles_h, self.n_tiles_w, self.max_sources)
+        latent_dims = (self.batch_size, self.n_tiles_h, self.n_tiles_w, self.max_sources, 1)
         b_flux = self._draw_truncated_pareto(
             self.star_flux_alpha, self.star_flux_min, self.star_flux_max, latent_dims
         )
-        total_flux = self.multiflux(star_ratios, b_flux)
```

Comment:
Can we remove the `multiflux` function completely? It looks like it isn't being used anywhere anymore.